app/src/main/res/values/strings.xml

```diff
@@ -116,4 +116,63 @@
   <string name="empty_forecast_list_server_error">No weather information available. The server is not returning valid data. Please check for an updated version of Sunshine.</string>
   <string name="empty_forecast_list_invalid_location">No weather information available. The location in settings is not recognized by the weather server.</string>
 
+  <!-- A11y -->
+  <string name="a11y_forecast">Forecast: <xliff:g id="condition">%1$s</xliff:g></string>
```

Comment:
the xliff tag also takes an `example` attribute I think, so you could provide an example of what the token's value to help translators understand the context.